### ISSUE TITLE
cant see if mod security installed 

### ISSUE THREAD

**Issue by GarethHards** [coreruleset/coreruleset#1981]
I would like to ask if someone can help me to check that the install is working ok with a basic rule please, is there code to use for my ubuntu 19.04 pi4 terminal that will tell me if the mod security is up and running, i had an issue earlier and renamed my apache2 server which caused me to have it not running, the apache2 is running fine and is the lastest version.

**Comment by dune73** [comment]
Apache tells you whether ModSec is loaded or not in the error.log during startup. You can then issue a 

```
curl http://localhost/index.html?exec=/bin/bash
```

This should trigger a ModSecurity alert in the Apache error.log.

If you have that, everything is fine and working.

For anything else, please refer to the Apache / ModSecurity tutorials at https://netnea.com.

I'm closing this now.
